tests/osaka/eip7692_eof_v1/eip3540_eof_v1/test_all_opcodes_in_container.py

```diff
@@ -382,3 +404,38 @@ def test_all_opcodes_stack_overflow(
         data=eof_code,
         expect_exception=exception,
     )
+
+
+@pytest.mark.parametrize(
+    "opcode",
+    sorted(push_opcodes),
+)
+@pytest.mark.parametrize(
+    "any_imm",
```

Comment:
any_imm name threw me off the scent. Maybe "truncate_whole" ("truncate_one_byte") verbosely

Reply:
At the risk of test bloat, should we test all lengths within a push?  It's over 500 tests but super easy to code.

Reply:
> At the risk of test bloat, should we test all lengths within a push? It's over 500 tests but super easy to code.

I'd say pass to this.